src/test/java/com/wojciechbarwinski/demo/epic_board_games_shop/exceptions/ProductsNotFoundExceptionTest.java

```diff
@@ -0,0 +1,26 @@
+package com.wojciechbarwinski.demo.epic_board_games_shop.exceptions;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ProductsNotFoundExceptionTest {
+
+
+    @Test
+    void testMessageGenerate(){
```

Comment:
Podziwiam rygor i dyscyplinę, ale testy, czy message'a wyjątku wygląda tak, a nie inaczej to już wyższy level xD

Reply:
Wywal ten test :D